packages/EasyErrorHandler/src/Bridge/Laravel/Provider/EasyErrorHandlerServiceProvider.php

```diff
@@ -35,20 +36,27 @@ public function register(): void
 
         $this->app->singleton(ErrorResponseFactoryInterface::class, ErrorResponseFactory::class);
 
+        $this->app->singleton(ErrorHandlerInterface::class, function (): ErrorHandlerInterface {
```

Comment:
We should recreate a new PR so we don't have the updates from the other PR here so we avoid any conflicts, especially because we don't need those changes to update the translation logic.

Reply:
I created a separate PR for this #438